src/app/api/clients/route.ts

```diff
@@ -0,0 +1,79 @@
+import { AirtableCRMProvider } from "../../../lib/crm/airtable";
+
+// ENV variables for now, pending CRM configuration feature
+const AIRTABLE_TEST_API_KEY = process.env.AIRTABLE_TEST_API_KEY || "";
+const AIRTABLE_TEST_BASE_ID = process.env.AIRTABLE_TEST_BASE_ID || "";
+const AIRTABLE_TEST_TABLE_ID = process.env.AIRTABLE_TEST_TABLE_ID || "";
+
+// TODO replace with the data from the airtable api
+const test_data = [
+  {
+    id: "12345",
+    client: "Lola M.",
+    phoneNumber: "+1234567890",
+    email: "judys@gmail.com",
+    lastContacted: "January 28, 2024, 5:26pm",
+  },
+  {
+    id: "163936",
+    client: "Judy S.",
+    phoneNumber: "+1234567890",
+    email: "judys@gmail.com",
+    lastContacted: "January 28, 2024, 5:26pm",
+  },
+  {
+    id: "4567",
+    client: "ABC",
+    phoneNumber: "+0987654321",
+    email: "abc@gmail.com",
+    lastContacted: "January 28, 2024, 5:26pm",
+  },
+  {
+    id: "192745",
+    client: "Judy S.",
+    phoneNumber: "+1234567890",
+    email: "john@gmail.com",
+    lastContacted: "January 28, 2024, 5:26pm",
+  },
+  {
+    id: "192745",
+    client: "Judy S.",
+    phoneNumber: "+1674567890",
+    email: "jane@gmail.com",
+    lastContacted: "January 28, 2024, 5:26pm",
+  },
+  {
+    id: "192745",
+    client: "Erin S.",
+    phoneNumber: "+1094567890",
+    email: "erin@gmail.com",
+    lastContacted: "January 28, 2024, 5:26pm",
+  },
+];
+
+export async function GET() {
+  const provider = new AirtableCRMProvider(
+    AIRTABLE_TEST_BASE_ID,
+    AIRTABLE_TEST_API_KEY
+  );
+
+  // TODO debug why the records are not being returned (only table info is being returned)
+  const table = provider.getTable(AIRTABLE_TEST_TABLE_ID);
+
+  // return new Response(JSON.stringify(table), {
+  //   headers: {
+  //     "content-type": "application/json",
+  //   },
+  // });
```

Comment:
When you do resolve the data fetching issue, make sure there's no commented code in the final push.